### PR TITLE
Drizzlepac notebook debug and fix : Drizzle_wfpc2 and Sky_matching

### PR DESCRIPTION
**Pull request by kyhuynh108**
Drizzlepac notebook fix to two notebooks: Drizzle_wfpc2 and sky_matching.

Drizzle_wfpc2: resolved "CalledProcessError".
fixed a couple of syntax errors that stems from new updates to AstroDrizzle.
Added a section that made the DQ array for the simulated fits, because Astrodrizzle could not run the simulated files without a DQ array.

Sky_Matching: 

Tweakreg cell: current parameters had trouble finding stars to match. I changed the parameters  and added a couple to help find matches between images.

tweakback cell:
changed drz_files; was calling the single files too, should only call the *0_drz.files. Also changed the input as the input has to be f + ‘[sci,1]’ to grab the header from the sci header
changed tweakback.tweakback to tweakback.apply_tweak, and added the correct parameters for it to work.

### CODE REVIEW

**Review by dulude — CHANGES_REQUESTED** [review]
I had previously excluded the sky_matching notebook from HTML generation because of the build errors. Thanks for fixing the issues. To remove the sky_matching notebook from the exclude list, please remove [line 46 in _config.yml ](https://github.com/kyhuynh108/hst_notebooks/blob/2f643b8bc1924ad3d311a15585463235bf25a399/_config.yml#L46) and [lines 33 and 34 in _toc.yml](https://github.com/kyhuynh108/hst_notebooks/blob/2f643b8bc1924ad3d311a15585463235bf25a399/_toc.yml#L33)

**Comment by kyhuynh108** [comment]
The edits to the exclude lists for sky_matching has been addressed.

**Review by dulude — APPROVED** [review]
Thanks. LGTM

File: _toc.yml [hunk 1]
```diff
@@ -30,8 +30,6 @@ parts:
       - file: notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb
       - file: notebooks/DrizzlePac/mask_satellite/mask_satellite.ipynb
       - file: notebooks/DrizzlePac/optimize_image_sampling/optimize_image_sampling.ipynb
-# Notebook excluded from build until build failures are fixed (See SPB-1160)
-#      - file: notebooks/DrizzlePac/sky_matching/sky_matching.ipynb
```
Comment (dulude) [inline _toc.yml:34]: Line 34 should simply be uncommented, not deleted.

**Pull request closed without merging**